Reference/Searching/Examine/Quick-Start/index-v8.md

```diff
@@ -0,0 +1,260 @@
+---
+versionFrom: 8.0.0
+versionTo: 8.0.0
+---
+
+# Quick start
+
+_This guide will help you get setup quickly using Examine with minimal configuration options. Umbraco ships Examine with 3 indexes: internal, external, and members. The internal index should not be used for searching when returning results on a public website because it includes content that has not been published yet. Instead you can use the external index to get up and running._
```

Comment:
```suggestion
_This guide will help you get set up quickly using Examine with minimal configuration options. Umbraco ships Examine with 3 indexes: internal, external, and members. The internal index should not be used for searching when returning results on a public website because it includes content that has not been published yet. Instead, you can use the external index to get up and running._
```